mypy/main.py

```diff
@@ -808,6 +809,15 @@ def add_invertible_flag(flag: str,
     code_group.add_argument(
         '--explicit-package-bases', action='store_true',
         help="Use current directory and MYPYPATH to determine module names of files passed")
+    code_group.add_argument(
+        "--exclude",
+        metavar="PATH",
+        default=options.exclude,
+        help=(
+            "Regex to match file names, directory names or paths to avoid checking. "
```

Comment:
Nit: Regx -> Regular expression (for extra clarity)

If default can't be overridden, remove "Defaults to ..." and maybe update to say "additional file names, ...".

Maybe explicitly add something about "recursing into directories" since this doesn't affect paths that refer to files.